spec/spec_helper.rb

```diff
@@ -29,7 +29,7 @@ def tmp_path( *paths )
 CarrierWave.root = public_path
 # DatabaseCleaner[:neo4j, connection: {type: :bolt, path: 'bolt://localhost:7006'}].strategy = :transaction
 
-neo4j_adaptor = Neo4j::Core::CypherSession::Adaptors::Bolt.new('bolt://localhost:7006', {ssl: false})
+neo4j_adaptor = Neo4j::Core::CypherSession::Adaptors::Bolt.new('bolt://localhost:7472', {ssl: false})
```

Comment:
Is this a standard port for testing databases, of some sort? It's oddly aligned with the HTTP/HTTPS ports, which is why I ask. If, on the other hand, it's just as arbitrary as 7006, I'm not sure I understand why this change is necessary.

Reply:
Yes, it is a default port.